scripts/deploy.ts

```diff
@@ -1,15 +1,18 @@
 import {ethers} from "hardhat";
 
 async function main() {
+  const MinValidatorCount = 1;
+  //Max uint32
+	const MaxValidatorCount = 4294967295;
+
   const [deployer] = await ethers.getSigners();
 
   console.log("Deploying contracts with the account:", deployer.address);
   console.log("Account balance:", (await deployer.getBalance()).toString());
 
   const StakingContractFactory = await ethers.getContractFactory("Staking");
-  const stakingContract = await StakingContractFactory.deploy({
-    gasLimit: '0x100000'
-  });
+
+  const stakingContract = await StakingContractFactory.deploy(MinValidatorCount, MaxValidatorCount);
```

Comment:
Why not have this values settable from the .env and only use defaults if they're not provided?

Reply:
Fixed 115588a7e5c6265e64aa2ce6755bf45a100913b9